desc/compute/_core.py

```diff
@@ -1540,6 +1540,242 @@ def _alpha_z(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="alpha_rt",
+    label="\\partial_{\\rho \\theta} \\alpha",
+    units="~",
+    units_long="None",
+    description="Field line label, derivative wrt poloidal and radial coordinate",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["theta_PEST_rt", "phi_t", "phi_rt", "iota", "iota_r"],
+)
+def _alpha_rt(params, transforms, profiles, data, **kwargs):
+    data["alpha_rt"] = (
+        data["theta_PEST_rt"]
+        - data["iota"] * data["phi_rt"]
+        - data["iota_r"] * data["phi_t"]
+    )
+    return data
+
+
+@register_compute_fun(
+    name="alpha_tz",
+    label="\\partial_{\\theta \\zeta} \\alpha",
+    units="~",
+    units_long="None",
+    description="Field line label, derivative wrt poloidal and toroidal coordinate",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["theta_PEST_tz", "phi_tz", "iota"],
+)
+def _alpha_tz(params, transforms, profiles, data, **kwargs):
+    data["alpha_tz"] = data["theta_PEST_tz"] - data["iota"] * data["phi_tz"]
+    return data
+
+
+@register_compute_fun(
+    name="alpha_rz",
+    label="\\partial_{\\rho \\zeta} \\alpha",
+    units="~",
+    units_long="None",
+    description="Field line label, derivative wrt radial and toroidal coordinates",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["theta_PEST_rz", "phi_z", "phi_rz", "iota", "iota_r"],
+)
+def _alpha_rz(params, transforms, profiles, data, **kwargs):
+    data["alpha_rz"] = (
+        data["theta_PEST_rz"]
+        - data["iota_r"] * data["phi_z"]
+        - data["iota"] * data["phi_rz"]
+    )
+    return data
+
+
+@register_compute_fun(
+    name="alpha_tt",
+    label="\\partial_{\\theta \\theta} \\alpha",
+    units="~",
+    units_long="None",
+    description="Field line label, second-order derivative wrt poloidal coordinate",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["phi_tt", "iota", "lambda_tt"],
+)
+def _alpha_tt(params, transforms, profiles, data, **kwargs):
+    data["alpha_tt"] = data["lambda_tt"] - data["iota"] * data["phi_tt"]
```

Comment:
Just for clarity can we add an alias `theta_PEST_tt = lambda_tt` and use `theta_PEST_tt` here? (same for the other new qtys in this file)

Reply:
This will soon be eliminated from this PR so do it somewhere else.